docs/FAQ/gui-faq-nb.md

```diff
@@ -0,0 +1,106 @@
+
+#### 9. Zonemaster og personvern <a name="q9"></a>
+Zonemaster lagrer all historikk. Det betyr at du kan gå tilbake og se på en test du gjorde for en uke siden og sammenligne med en test du nettopp har gjort.
+
+Zonemaster er tilgjengelig for alle, dermed er det også mulig for alle å teste ditt domene og også se historikk for domenet. Det er imidlertid ingen som kan se hvem som har gjort en test. Det eneste som er logget, er tidspunktet da testen ble utført.
+
+#### 10. Hvorfor kan jeg ikke teste mitt domene? <a name="q10"></a>
+Hvis vi ser bort fra situasjonen der domenet ikke eksisterer er det to andre muligheter:
+  - For å beskytte motoren mot flere identiske tester på samme domene fra en og
+    samme IP-adresse er det en forsinkelse på 5 minutter mellom
+    identiske påfølgende tester. Dette betyr at du bare kan teste
+    samme domene en gang hvert 5. minutt, hvis du prøver å teste det igjen
+    innen 5 minutter vil det forrige resultatet vises i stedet.
+  - Fordi Zonemaster ble laget for å sjekke domener eller DNS-soner, som zonemaster.net, og
+    ikke vertsnavn i et domene (sone), som www.zonemaster.net, vil Zonemaster gjøre det
+    rapporter en feil hvis du prøver å teste et vertsnavn i stedet for et domene som samsvarer med en DNS-sone.
+
+#### 11. What kind of queries does Zonemaster generate? <a name="q11"></a>
+Zonemaster vil generere forskjellige typer spørringer avhengig av hvordan DNS-serverne svarer.
+
+For å få en full visning av hvilke spørsmål og resultater som genereres, kan du kjøre CLI-grensesnittet og velg full utdata. For å kunne kjøre CLI-grensesnittet må du laste ned pakke og installer den.
+Resultatet fra CLI-verktøyet er ganske tungt teknisk så med mindre du er interessert i bit og byte, vil du kanskje hoppe over dette trinnet.
+
+#### 12. Hva er et udelegert test? <a name="q12"></a>
+Du kan sjekke domener og navnetjenere som ikke er publisert i DNS, eller endringer av navnetjenere for et domene før endringen er publisert. Hvis testen er feilfri, kan du regne med at navnetjeneren vil svare på spørringer om domenet. Det kan imidlertid fortsatt være feil i soneinformasjonen som denne testen ikke kjenner til.
+
+#### 13. Hvordan kan jeg teste revers-soner med Zonemaster? <a name="q13"></a>
+Et reversoppslag kan brukes til å finne ut hvilket domenenavn som er knyttet til en IP-adresse.
+
+For en IPv4-adresse må du først finne nettverksadressen til systemet. Denne ender oftest med en 0 (null). Du sletter siste 0 (null), deretter endrer du rekkefølgen på sifrene du har mottatt og legger til suffikset in-addr.arpa. Du gjør det samme med en IPv6-adresse, men legger til suffikset ip6.arpa.
+
+Eksempel 1 - reversoppslag for et IPv4-nett:
+Nettadressen er 194.98.30.0 og gir reversoppslagssonen 30.98.194.in-addr.arpa.
```

Comment:
Just a note here. Is this an intentional line break ? Because I don't know how this line break will be rendered in the GUI. What I know is that it is not rendered in the [Github file view (b0a93ea)](https://github.com/zonemaster/zonemaster-gui/blob/b0a93ea434d0fab6e7e017eb0f9db349284fe057/docs/FAQ/gui-faq-nb.md#q13)